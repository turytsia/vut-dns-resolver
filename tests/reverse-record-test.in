-r -t -x -s kazi.fit.vutbr.cz 147.229.9.23