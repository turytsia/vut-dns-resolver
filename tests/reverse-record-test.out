Authoritative: Yes, Recursive: Yes, Truncated: No
Question section (1)
 23.9.229.147.in-addr.arpa., PTR, IN
Answer section (1)
 23.9.229.147.in-addr.arpa., PTR, IN, 0, www.fit.vutbr.cz.
Authority section (0)
Additional section (0)
